src/types.js

```diff
@@ -58,5 +58,13 @@ export const TYPES = {
   inAppPurchasesPolicy: {
     name: 'In App Purchases Policy',
     fileName: 'in_app_purchases_policy'
+  },
+  userConsentPolicy: {
```

Comment:
Sounds legit to be added:

- Writer: service provider or parent organization.
- Audience: client companies.
- Object: proving end user consent.